src/NDItk/multigroup/FluxWeights/src/read.hpp

```diff
@@ -6,11 +6,11 @@
  *
  *  @param[in,out] iter     an iterator to the current position in the input
  *  @param[in,out] end      an iterator to the end of the input
- *  @param[in,out] groups   the number of weight values to be read
+ *  @param[in,out] number   the number of weight values
```

Comment:
Why the change? The number of weight values is also the number of groups.

Reply:
Less changes when i copy/paste that line ;-)